flink-table/flink-table-runtime/src/test/java/org/apache/flink/table/runtime/functions/table/fullcache/inputformat/InputFormatCacheLoaderTest.java

```diff
@@ -178,12 +184,16 @@ private void assertCacheContent(Map<RowData, Collection<RowData>> actual) {
                         assertThat(rows).containsExactlyInAnyOrderElementsOf(actual.get(key)));
     }
 
+    private void run(CacheLoader cacheLoader) {
```

Comment:
```suggestion
    private void reloadSynchronously(CacheLoader cacheLoader) {
```
nit: that would be a better description of what this method is doing.

Reply:
Fixed